applications/StructuralMechanicsApplication/custom_elements/small_displacement_mixed_volumetric_strain_element.h

```diff
@@ -522,6 +519,9 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) SmallDisplacementMixedVolumet
     ///@name Member Variables
     ///@{
 
+    Matrix mAnisotropyTensor;
+    Matrix mInverseAnisotropyTensor;
```

Comment:
Question, is not better to invert on running time the tensor instead of storing it (with the memmory cost)?

Reply:
We had this same discussion... I'm not sure at all.

Reply:
I think the general rule is to only store what cannot be recomputed 
At least that’s how I understood Riccardo

Reply:
I agree on this one

Reply:
I mean, it may not be FASTER to invert it on the flight, however storing it implies storing 36 doubles per node.

btw, do we still need the T^-1 with the latest formulation? i thought that the term that needed it is out

Reply:
I'll invert it on the fly then. 

It is still required as we substitute the symgrad(u) by the enriched strain in the momentum equation.

Reply:
Guys, having a look to the formulation and after trying to efficiently implement it, I think that in this case it is worth it to store it. 

The point is that we also need the inv(T) to calculate the equivalent strain, which is done in the kinematics calculation, which has to be done inside each Gauss pt. As an alternative, we could change the definition of the `CalculateKinematicVariables` method. I don't like this last option as it would differ from the rest of elements in the structural mechanics application.